scripts/local_testnet/docker-compose.yml

```diff
@@ -0,0 +1,95 @@
+version: '3'
+services:
+  ganache:
+    build: .
+    network_mode: "host"
+    entrypoint: "./ganache_test_node.sh"
+  
+  boot:
+    build: .
+    network_mode: "host"
+    volumes:
+      - data:/root/.lighthouse
+    depends_on: 
+      - ganache
+    entrypoint: bash -c "sleep 5 && ./setup.sh && ./bootnode.sh"
+
+  ##################################################
+  beacon1:
+    build: .
+    network_mode: "host"
+    volumes:
+      - data:/root/.lighthouse
+    depends_on: 
+      - boot
+    entrypoint: bash -c "sleep 25 && ./beacon_node.sh $HOME/.lighthouse/local-testnet/node_1 9000 8000"
```

Comment:
For me `$HOME` was picking up my local machine's home directory as opposed to `$HOME` inside the docker container so the datadir wasn't being shared between containers and the validator clients couldn't find the generated keys.  So replacing `$HOME` with `/root` everywhere in this file fixed it for me.